dockerfiles/rocky9.2-10-cpu/Dockerfile

```diff
@@ -0,0 +1,6 @@
+# syntax=docker/dockerfile:experimental
+FROM rockylinux:9.2
+
```

Comment:
do we not still need dnf upgrate / upgrade here? this still needs to install pandoc/libreoffice and other deps right?

Reply:
Since this is all in a single layer, you probably could do the these steps in the first script and cleanup in the last

Reply:
The `base.sh` should be running that for us.

Reply:
It was only needed in the python one because it doesn't call any dependency scripts with that included, since it's only installing python

Reply:
ohh derp, yep, see that now. thanks!